applications/nrf5340_audio/doc/user_interface.rst

```diff
@@ -84,77 +89,90 @@ LEDs
+|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+------------------+------------+
+|                          | Blinking on the CIS gateway: Kit is streaming to a headset.                                               | :ref:`nrf53_audio_unicast_client_app`       |         Y        |      Y     |
+|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+------------------+------------+
+|                          | Blinking on the BIS gateway: Kit has started broadcasting audio.                                          | :ref:`nrf53_audio_broadcast_source_app`     |                  |            |
++--------------------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+------------------+------------+
+| **LED2**                 | Off - Sync not achieved.                                                                                  | All                                         |         Y        |      Y     |
+|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+------------------+------------+
+|                          | Solid green - Sync achieved (both drift and presentation compensation are in the ``LOCKED`` state).       | * :ref:`nrf53_audio_broadcast_sink_app`     |         Y        |      Y     |
+|                          |                                                                                                           | * :ref:`nrf53_audio_unicast_server_app`     |                  |            |
++--------------------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+------------------+------------+
+| **LED3**                 | Blinking green - The nRF5340 application core is running.                                                 | All                                         |         Y        |      Y     |
++--------------------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+------------------+------------+
+| **LED5**                 | Off - No PC connection available.                                                                         | All                                         |         N        |      Y     |
```

Comment:
This LED also exists on the 5340 DK. Discuss as above if we should write "IFMCU" in the Audio DK column and "LED5" in the vanilla DK column.

Also, this info is not that crucial. @greg-fer, do we usually document this IFMCU/Segger behavior on each DK?

Reply:
There are no guidelines about not documenting it. User Interface should document the behavior of the LEDs and buttons for the purpose of running and testing the application. If it needs mentioning SEGGER components, we can do it.